src/python-frontend/python_converter.cpp

```diff
@@ -0,0 +1,258 @@
+#include "python-frontend/python_converter.h"
+#include "util/std_code.h"
+#include "util/c_types.h"
+#include "util/arith_tools.h"
+#include "util/expr_util.h"
+#include "util/message.h"
+
+#include <fstream>
+#include <unordered_map>
+
+using json = nlohmann::json;
+
+const char *json_filename = "/tmp/ast.json";
+
+std::unordered_map<std::string, std::string> operator_map = {
+  {"Add", "+"},
+  {"Sub", "-"},
+  {"Mult", "*"},
+  {"Div", "/"},
+  {"BitOr", "bitor"},
+  {"BitAnd", "bitand"},
+  {"BitXor", "bitxor"},
+  {"Invert", "bitnot"},
+  {"LShift", "shl"},
+  {"RShift", "lsr"},
+  {"USub", "unary-"}};
+
+enum class StatementType
+{
+  VARIABLE_ASSIGN,
+  FUNC_DEFINITION,
+  UNKNOWN,
+};
+
+enum class ExpressionType
+{
+  UNARY_OPERATION,
+  BINARY_OPERATION,
+  LITERAL,
+  UNKNOWN,
+};
+
+StatementType get_statement_type(const json &element)
```

Comment:
Can we make this static? No need to export and potentially confuse the linker in case there is another function with the same signature somewhere else. Also applies to the non-member functions below.